src/stream-tcp.c

```diff
@@ -5799,6 +5801,14 @@ int StreamTcpBypassEnabled(void)
     return stream_config.bypass;
 }
 
+void StreamTcpSetReassemblyDepth(TcpSession *ssn, uint32_t size)
+{
+    ssn->server.reassembly_depth = size;
```

Comment:
We have added two values because it would have been required to modify the API in TCP streaming. Furthermore it could be interesting later to dissociate the value.